.github/workflows/build.yaml

```
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: 8
+          java-version: 17
```

```suggestion
          java-version: 21
```

With the latest APG, we can run Java 21 on CI.